handlers/additem.js

```diff
@@ -152,12 +155,16 @@ const notifyAdditemSuccess = async function (query, itemName, order_id) {
     const replyListenerId = await bot.onReplyToMessage(query.message.chat.id, query.message.message_id, async (replymsg) => {
         try {
             if (await queries.hasOrder(replymsg.reply_to_message.message_id)) {
-                await queries.addRemark({
-                    remarks: replymsg.text,
-                    message_id: replymsg.reply_to_message.message_id,
-                });
-                messenger.send(replymsg.chat.id, 'Remark added successfully!');
-                queries.refreshLiveCountMessage(await queries.getChatIdFromOrderId(order_id))
+                if (replymsg.text.length > 60) {
+                    messenger.send(replymsg.chat.id, REMARK_TOO_LONG_RESPONSE);
```

Comment:
hmm is there a reason you don't add a return statement here? this would fit the guard clause pattern, and allow the "happy path" code to not be indented in the else block

Reply:
Done, thanks for the review!